executable-monitor/action.yml

```diff
@@ -14,7 +14,11 @@ inputs:
   timeout-seconds:
     description: 'Maximum amount of time to run the executable. Default is 600.'
     required: false
-    default: 600
+    default: 300
+  retry-attempts:
+    description: 'Number of times to re-launch the binary to check for success.'
+    required: false
+    default: 1
```

Comment:
Why is this default now 1 retry attempt? Was this being retried before? This also seems like a strange default value given the python script uses 0 as it's default.